src/transformers/models/mistral/modeling_mistral.py

```diff
@@ -820,7 +910,7 @@ def __init__(self, config: MistralConfig):
         self.layers = nn.ModuleList(
             [MistralDecoderLayer(config, layer_idx) for layer_idx in range(config.num_hidden_layers)]
         )
-        self._use_flash_attention_2 = config._attn_implementation == "flash_attention_2"
```

Comment:
Any reason this has been removed? Shall we keep it for consistency with llama: https://github.com/huggingface/transformers/blob/main/src/transformers/models/llama/modeling_llama.py#L961 ? I found it much more readable to have `_use_flash_attention_2` instead of checking `self._attn_implementation == "flash_attention_2"` each time

Reply:
No because we need to have one for sdpa as well which does not make sense